config.yaml.example

```diff
@@ -9,6 +9,9 @@ twtfile: ~/public_html/twtxt.txt
 # add User-Agent header with nick and program version when fetching feed (default true)
 #discloseidentity: true
 
+# include your own posts in the timeline
+# includeyourself: true
```

Comment:
I agree that this should be the default. The default value of a bool in Go is false though. Maybe you should add a true default to the config struct in main.

Reply:
I mixed up of course! I think the default should be false, thus not changing the current behaviour. So putting something in the config struct wont be needed. Just change this to false.

Reply:
Done